src/main/java/io/quarkus/registry/app/model/ExtensionRelease.java

```diff
@@ -105,9 +108,14 @@ public static Optional<ExtensionRelease> findByGAV(String groupId, String artifa
     }
 
     public static List<ExtensionRelease> findNonPlatformExtensions(String quarkusCore) {
+        DefaultArtifactVersion artifactVersion = new DefaultArtifactVersion(quarkusCore);
+        String minimumVersion = artifactVersion.getMajorVersion() + ".0.0.Final";
```

Comment:
The minimum version should be `Alpha`, since `3.0.0.Alpha` is less than `3.0.0.Final`.